### PR TITLE
Parity DCR DOM tags with frontend

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Changes some DOM tags to better match frontend

## Why?
for both google indexing and spaceFinder (which uses DOM tags to determine where to place elements in a page)

### CODE REVIEW

File: src/web/components/elements/BlockquoteBlockComponent.tsx [hunk 1]
```diff
@@ -47,6 +41,6 @@ export const BlockquoteBlockComponent: React.FC<Props> = ({
                 `}
                 tagName="blockquote"
```
Comment (gtrufitt) [inline src/web/components/elements/BlockquoteBlockComponent.tsx:42]: You're gonna double blockquote here.

File: src/web/lib/ArticleRenderer.tsx [hunk 2]
```diff
@@ -278,14 +278,14 @@ export const ArticleRenderer: React.FC<{
                     );
                 case 'model.dotcomrendering.pageElements.RichLinkBlockElement':
                     return (
-                        <div key={i} id={`rich-link-${i}`}>
+                        <aside key={i} id={`rich-link-${i}`}>
```
Comment (gtrufitt) [inline src/web/lib/ArticleRenderer.tsx:281]: Might it make sense to move this `aside` along with the key and id generation into the component itself?

**Review by gtrufitt — APPROVED** [review]
Yay to semantics! and parity! Couple of suggestions but +1 when you've looked at them 👍

**Comment by oliverlloyd** [comment]
@liywjl do we still want to move forward with this PR?

**Comment by liywjl** [comment]
I thought this had an impact on Ads and space finder. It seemed not to in the end, and I had some challenges with lifting dangerously insterted DOM elements to fit the syntax. It might be worth revisiting one day, but is fairly low impact

**Pull request closed without merging**